entries/d/dollargeneral.com.json

```
@@ -0,0 +1,12 @@
+{
+  "Dollar General": {
+    "domain": "dollargeneral.com",
+    "img": "dollargeneral.com.png",
+    "tfa": [
+      "sms"
```

```suggestion
      "sms",
      "call"
```

Your screenshot says that calls might also be used.

---

Good catch